core/src/main/resources/META-INF/spring/cse.bean.xml

```diff
@@ -36,5 +36,6 @@
 
   <bean id="cse.executor.groupThreadPool" class="org.apache.servicecomb.core.executor.FixedThreadExecutor"></bean>
   <alias name="cse.executor.groupThreadPool" alias="cse.executor.default"/>
+  <alias name="cse.executor.groupThreadPool" alias="servicecomb.executor.groupThreadPool"/>
   <bean id="cse.executor.reactive" class="org.apache.servicecomb.core.executor.ReactiveExecutor"></bean>
```

Comment:
cse.executor.reactive also need a alias?

Reply:
an alias has been added for cse.executor.reactive